dsc/decisionMaker.md

```diff
@@ -6,13 +6,13 @@ title:  Desired State Configuration Overview for Decision Makers
 
 # Desired State Configuration Overview for Decision Makers
 
-This document describes the business benefits of using PowerShell Desired State Configuration (DSC). It is not a technical guide.
+This document describes the business benefits of using Windows PowerShell Desired State Configuration (DSC). It is not a technical guide.
 
 ## What Is Desired State Configuration?
 
-Windows PowerShell Desired State Configuration (DSC) is a configuration management platform built into Windows that is based on open standards. DSC is flexible enough to function reliably and consistently in each stage of the deployment lifecycle (development, test, pre-production, production), as well as during scale-out.
+PowerShell Desired State Configuration is a configuration management platform built into Windows that is based on open standards. DSC is flexible enough to function reliably and consistently in each stage of the deployment lifecycle (development, test, pre-production, production), as well as during scale-out.
 
-DSC centers around "[configurations](https://msdn.microsoft.com/powershell/dsc/configurations)".
+DSC centers around "[configurations](https://msdn.microsoft.com/powershell/dsc/configurations)."
```

Comment:
Not a punctuation expert but IMO it should stay as it is as quotes are around `configurations`, not the whole sentence.

Reply:
@robdy - In US English, it is common to put the comma or period inside the quotes if the quotation is at the end of the sentence.  I'd probably actually remove the quotations completely as I don't believe they are needed or add value in context.

@ulysesramos - Can you remove the quotes for this line?  I believe the link already provides enough emphasis.